betelgeuse.py

```diff
@@ -853,6 +853,12 @@ def create_xml_property(name, value):
     is_flag=True,
 )
 @click.option(
+    '--response-property',
+    nargs=2,
+    help='When defined, the impoter will mark all responses with the selector.'
+    'The format is "--response-property  property_key property_value".',
```

Comment:
@elyezer I disagree about this being 'personal taste'. I understand that

> nargs is a valid feature that even argparse offers

but in terms of usability, having too many values, specially if they're not associated with an argument, can lead to confusion and poor user experience imho.